packages/react/src/components/dialog/__snapshots__/Dialog.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Dialog /> spec renders the component 1`] = `<DocumentFragment />`;
```

Comment:
This feels like surprisingly short snapshot? One could expect that the size of Dialog snapshot would be bigger? Hmm.

Reply:
Yes, there are some problems with how the React Testing library's render works with portals and snapshots. 
I have not yet figured out how to render without following warnings: 
```
 Warning: render(): Rendering components directly into document.body is discouraged, since its children are often manipulated by third-party scripts and browser extensions. This may lead to subtle reconciliation issues. Try rendering into a container element created for your app.
 ```